src/main/scala/com/amazon/deequ/checks/Check.scala

```diff
@@ -672,7 +672,7 @@ case class Check(
       hint: Option[String] = None)
     : CheckWithLastConstraintFilterable = {
 
-    satisfies(s"$column >= 0", s"$column is non-negative", hint = hint)
+    satisfies(s"COALESCE($column, 0.0) >= 0", s"$column is non-negative", hint = hint)
```

Comment:
Can you add comments that describe why you are using COALESCE? That will be helpful for other readers of the code who haven't experienced the problems with missing values.

Reply:
Yes, good point, will add.